service/src/main/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelper.java

```diff
@@ -54,6 +55,12 @@ public static String formatCaseUrl(String uiBaseUrl, Long caseId, String tab) {
         return isBlank(tab) ? caseUrl : String.format("%s#%s", caseUrl, tab);
     }
 
+    public static String buildCallout(final CaseData caseData) {
```

Comment:
Do we need to pass everything here? Can it not just be the specific parameters?

Reply:
It was here originally, I'm just moving it to a more accessible place.

Reply:
i think passing entire case makes this api much easier to extends, if for some reason new things needs to be added do callout, method signature does not need to change. Beside that it encapsulates better, From client perspetive it is like give me callout, i dont care how you do this and what is required, it is your job to know. If we pass particular params, client needs to have much more knowledge what is need to build callout